doc/api-extensions.md

```diff
@@ -1636,3 +1636,6 @@ client authentication.
 
 ## images\_target\_project
 Adds ability to copy image to a project different from the source.
+
+## cluster\_migration\_inconsistent\_copy
+Adds ability to use the `--allow-inconsistent` flag when copying an instance between cluster members.
```

Comment:
This is just a nit, but really we shouldn't mention the `lxc` CLI flags in anything to do with the API (extensions, error messages etc) because the consumer of the API may not be the `lxc` CLI developer, but a developer of a different tool. One way `--allow-inconsistent` command flag doesn't exist. In this case I think it would be better to reference the field that has been added rather than the flag.

Reply:
Ok yeah that makes sense.

Reply:
Also I've just realised that we need to check the server has this extension when making any of these calls from the client.

Reply:
Yes, good thinking! :)